modules/rostests/winetests/ws2_32/sock.c

```diff
@@ -11620,9 +11629,25 @@ START_TEST( sock )
     test_GetAddrInfoW();
     test_GetAddrInfoExW();
     test_getaddrinfo();
+
+#ifdef __REACTOS__
+    if (!winetest_interactive)
+    {
+        skip("WSPAcceptEx(), WSPConnectEx() and WSPDisconnectEx() are UNIMPLEMENTED on ReactOS\n");
+        skip("Skipping tests due to hang. See ROSTESTS-385\n");
```

Comment:
I think this second line belongs to commit message, or at least a comment. At both places.
```suggestion
```

Reply:
I think this second line belongs to commit message, or at least a comment. At both places.
```suggestion
```